src/main/java/network/brightspots/rcv/ContestConfig.java

```diff
@@ -335,6 +334,18 @@ private void validateCandidates() {
         isValid = false;
         Logger.log(
             Level.SEVERE, "Duplicate candidate names are not allowed: %s", candidate.getName());
+      } else if (TallyTransfers.candidateStringIsReserved(candidate.getName())) {
```

Comment:
Cool check. Dumb question, but we don't want/need to do something similar for other key words like "UWI", "overvote", etc. right?

Reply:
Not a dumb question. We should do this, actually.